accounting/accounting/doctype/stock_entry/test_stock_entry.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2023, Akhil and Contributors
+# See license.txt
+
+import random
+import string
+
+import frappe
+from frappe.tests.utils import FrappeTestCase
+
+
+class TestStockEntry(FrappeTestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.incoming_warehouse_name = "".join(random.choices(string.ascii_letters, k=10))
+        self.outgoing_warehouse_name = "".join(random.choices(string.ascii_letters, k=10))
+        self.main_warehouse_name = "".join(random.choices(string.ascii_letters, k=10))
```

Comment:
There are some helper functions for this. 

frappe.generate_hash(), `random_string`

---

We should have some better API for faster fixture generation on FrappeTestCase.